components/event-publisher-proxy/cmd/event-publisher-proxy/main.go

```diff
@@ -14,7 +16,7 @@ import (
 
 func main() {
 	logger := logrus.New()
-
+	options := options.ParseArgs()
```

Comment:
Please change it to be `opts := options.ParseArgs()`, because `options` is already used as an import alias.

Reply:
Changed